src/json-formatter.js

```diff
@@ -31,13 +31,55 @@ angular.module('jsonFormatter', ['RecursionHelper'])
     return typeof object;
   }
 
+  function isObject(object) {
+    return object && typeof object === 'object';
+  }
+
+  function isArray(object) {
+    return Array.isArray(object)
+  }
+
+  function parseValue (object, value) {
+    var type = getType(object);
+    if (type === 'null') {
+      return 'null';
+    }
+    if (type === 'undefined') {
+      return 'undefined';
+    }
+    if (type === 'string') {
+      value = '"' + escapeString(value) + '"';
+    }
+    if (type === 'function'){
+
+      // Remove content of the function
+      return object.toString()
+          .replace(/[\r\n]/g, '')
+          .replace(/\{.*\}/, '') + '{ ... }';
+
+    }
+    return value;
+  }
+
+  function parseThumbnail(object) {
```

Comment:
This function name is not describing what it's doing exactly. Please use a name like `getPreview`